admin/create-theme/theme-styles.php

```diff
@@ -29,7 +30,7 @@ public static function build_child_style_css( $theme ) {
 Version: 0.0.1
 License: GNU General Public License v2 or later
 License URI: http://www.gnu.org/licenses/gpl-2.0.html
-Template: {$template}
+{$template_text}
```

Comment:
It still keeps a blank line

<img width="436" alt="image" src="https://user-images.githubusercontent.com/1935113/231718975-45a67add-3d62-4e33-bd94-68876854251f.png">

Reply:
fixed in the latest commit.